src/main/java/seedu/recipe/logic/commands/FavouriteCommand.java

```diff
@@ -36,31 +39,66 @@ public FavouriteCommand(Index[] targetIndex) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Recipe> lastShownList = model.getFilteredRecipeList();
-        StringBuilder sb = new StringBuilder().append("Added ");
+        List<String> successfulFavouritesList = new ArrayList<>();
```

Comment:
perhaps the naming could be slightly more different or javadoc comments could be added to clarify, because when I first read it I didn't really get the difference between the two (but maybe it's just me)

Reply:
Fixed in latest PR! I hope the new naming makes more sense (see latest PR)